src/main/java/org/generationcp/middleware/api/breedingmethod/BreedingMethodService.java

```diff
@@ -10,4 +10,6 @@
 	BreedingMethodDTO getBreedingMethod(Integer breedingMethodDbId);
 
 	List<BreedingMethodDTO> getBreedingMethods(BreedingMethodSearchRequest methodSearchRequest);
```

Comment:
I think you can just use `getBreedingMethods(BreedingMethodSearchRequest methodSearchRequest)` without any search filter